download.md

```diff
@@ -21,39 +21,61 @@ limitations under the License.
+    ([pgp](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0.tgz.asc),
+     [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0.tgz.md5),
+     [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0.tgz.sha))
 
-    * Binary package:
-    <a style="cursor:pointer" onclick="ga('send', 'event', 'download', 'zeppelin-bin', '0.5.6-incubating'); window.location.href='http://www.apache.org/dyn/closer.cgi/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz'">zeppelin-0.5.6-incubating-bin-all.tgz</a>
-    ([pgp](https://www.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz.asc),
-     [md5](https://www.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz.md5),
-     [sha](https://www.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz.sha))
+    * Binary package with all interpreters:
+    <a style="cursor:pointer" onclick="ga('send', 'event', 'download', 'zeppelin-bin-all', '0.6.0'); window.location.href='http://www.apache.org/dyn/closer.cgi/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-all.tgz'">zeppelin-0.6.0-bin-all.tgz</a>
+    ([pgp](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-all.tgz.asc),
+     [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-all.tgz.md5),
+     [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-all.tgz.sha))
+
+    * Binary package with Spark interpreter and interpreter net-install script ([interpreter installation guide](../../docs/0.6.0/manual/interpreterinstallation.html)):
+    <a style="cursor:pointer" onclick="ga('send', 'event', 'download', 'zeppelin-bin-netinst', '0.6.0'); window.location.href='http://www.apache.org/dyn/closer.cgi/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz'">zeppelin-0.6.0-bin-netinst.tgz</a>
+    ([pgp](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.asc),
+     [md5](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.md5),
+     [sha](https://www.apache.org/dist/zeppelin/zeppelin-0.6.0/zeppelin-0.6.0-bin-netinst.tgz.sha))
 
 
 
 ### Verify the integrity of the files
 
-It is essential that you [verify](https://www.apache.org/info/verification.html) the integrity of the downloaded files using the PGP or MD5 signatures. This signature should be matched against the [KEYS](https://www.apache.org/dist/incubator/zeppelin/KEYS) file.
+It is essential that you [verify](https://www.apache.org/info/verification.html) the integrity of the downloaded files using the PGP or MD5 signatures. This signature should be matched against the [KEYS](https://www.apache.org/dist/zeppelin/KEYS) file.
 
```

Comment:
@bzz Sorry, I missed mentioning about KEYS file link. I just updated description.

Actually I already added my key and located it under https://dist.apache.org/repos/dist/release/zeppelin/KEYS

As far as I understand from [Apache release process doc](http://www.apache.org/dev/release-publishing.html#distribution_dist), all contents of https://dist.apache.org/repos/dist/release/zeppelin will be pushed to https://www.apache.org/dist/zeppelin once svnpubsub set.

So I believe [INFRA-12205](https://issues.apache.org/jira/browse/INFRA-12205) will also solve this issue. Let me know if there is any further action I should take :)